onward/app/controllers/RecommendedContentCardController.scala

```diff
@@ -0,0 +1,48 @@
+package controllers
+
+import com.gu.contentapi.client.model.v1.ItemResponse
+import contentapi.ContentApiClient
+import layout.ContentCard
+import model.ApplicationContext
+import play.api.mvc.{Action, RequestHeader}
+
+import scala.concurrent.Future
+
+class RecommendedContentCardController(contentApiClient: ContentApiClient)(implicit context: ApplicationContext) extends RenderTemplateController(contentApiClient) {
+
+  def render(path: String) = Action.async { implicit request =>
+    makeContentCardHtml(lookup(path)) map { maybeHtml =>
+      maybeHtml match {
+        case Some(html) => renderContent(html, html)
+        case None => NotFound
+      }
```

Comment:
this can be simplified to:

    makeContentCardHtml(lookup(path)) map {
      case Some(html) => renderContent(html, html)
      case None => NotFound
    }